google/cloud/security/inventory/api_map.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Map of the APIs needed by the inventory pipelines."""
```

Comment:
Ack on making this map more handled by the code.  One problem here is that the class name need to be more consistent with the module name.

I also did think about combining this with the requirements map, but that would complicate the requirements map, by adding another layer.  So, it's simpler with 2 maps.